filament/src/details/Scene.h

```diff
@@ -180,6 +183,8 @@ class FScene : public Scene {
     tsl::robin_set<utils::Entity> mEntities;
     RenderableSoa mRenderableData;
     LightSoa mLightData;
+    UniformBuffer mUniformBuffer;
```

Comment:
Maybe add a one-line comment explaining what data is held in this uniform buffer.